cfg/cfg.go

```diff
@@ -115,6 +117,13 @@ func init() {
 		"Output events that are usually sent Che master instead of regular logs to imitate what a user can see."+
 			"`false` by default. Needed for testing and debugging purposes",
 	)
+	flag.BoolVar(
+		&UseLocalhostInPluginUrls,
+		"use-localhost-in-plugin-urls",
+		true,
+		"This configures the broker to use the `localhost` name instead of the Kubernetes service name to build Theia or VSCode plugin endpoint URL."+
+			"`true` by default since until now all remote VSCode or Theia plugin containers are started on the same POD as the Theia IDE container",
```

Comment:
```suggestion
			"`true` by default since until now all remote VS Code or Theia plugin containers are started on the same POD as the Theia IDE container",
```